server/src/main/scala/com/cloudera/livy/server/ApiVersioningSupport.scala

```diff
@@ -0,0 +1,98 @@
+package com.cloudera.livy.server
```

Comment:
hi @tc0312,

You're missing copyright headers in some files. I have a pending style check to make sure they're there, but in the meantime can you add them to this PR?